samples/boards/gd32/spi_gd25q16b/README.rst

```diff
@@ -0,0 +1,29 @@
+.. _gd32_spi_gd25q16b:
```

Comment:
note: you need to implement a flash driver instead (same as EEPROM for I2C)

Reply:
use spi_flash sample instead.
done.